include/SDL3/SDL_filesystem.h

```diff
@@ -363,6 +364,83 @@ extern SDL_DECLSPEC SDL_bool SDLCALL SDL_GetPathInfo(const char *path, SDL_PathI
  */
 extern SDL_DECLSPEC char ** SDLCALL SDL_GlobDirectory(const char *path, const char *pattern, SDL_GlobFlags flags, int *count);
 
+/* Carried over from SDL_iostream.h */
+typedef struct SDL_IOStream SDL_IOStream;
```

Comment:
This causes type redefinition errors:
```
In file included from include/SDL3/SDL.h:44,
                 from test.c:1:
include/SDL3/SDL_filesystem.h:414: error: redefinition of typedef ‘SDL_IOStream’
include/SDL3/SDL_iostream.h:182: note: previous declaration of ‘SDL_IOStream’ was here
```

Suggest including the header, instead:
```diff
diff --git a/include/SDL3/SDL_filesystem.h b/include/SDL3/SDL_filesystem.h
--- a/include/SDL3/SDL_filesystem.h
+++ b/include/SDL3/SDL_filesystem.h
@@ -32,2 +32,3 @@
 #include <SDL3/SDL_error.h>
+#include <SDL3/SDL_iostream.h>
 
@@ -412,5 +413,2 @@ extern SDL_DECLSPEC char ** SDLCALL SDL_GlobDirectory(const char *path, const ch
 
-/* Carried over from SDL_iostream.h */
-typedef struct SDL_IOStream SDL_IOStream;
-
 /**
```

Reply:
How is this resolved?

Reply:
Have I forgotten to commit this before pushing? Let me double-check...

Reply:
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   ../include/SDL3/SDL_filesystem.h
```

Must've forgotten to `git add`, my bad! It's pushed now.